frontend/packages/kubevirt-plugin/src/components/form/helper/url-source-help.tsx

```diff
@@ -22,7 +22,7 @@ export const URLSourceHelp: React.FC<URLSourceHelpProps> = ({ baseImageName }) =
           RHEL download page
         </a>
       </strong>{' '}
-      (requires login) and copy the download link URL of the KVM guest image
+      (requires login) and copy the download link URL of the KVM guest image (expires quickly)
```

Comment:
maybe describe the link as temporary ?
```suggestion
      (requires login) and copy the temporary download link URL of the KVM guest image
```